.github/scripts/update-perf.sh

```diff
@@ -19,66 +19,47 @@ start_template='<!-- START_%s_RESULTS_TABLE -->'
 end_template='<!-- END_%s_RESULTS_TABLE -->'
 
 if [ "$#" -ne 3 ]; then
-  echo "Must supply version and type, eg update-perf.sh 'v0.39.0' 'latency|correctness' '/path/to/file'"
+  echo "usage: update-perf.sh <version> <perf-path> <correctness-path>"
   exit 1;
 fi
 
 version="$1"
-type="$2"
-new_table="$3"
+new_perf="$2"
+new_correctness="$3"
 
 if [[ "$OSTYPE" == "linux-gnu"* ]]; then
   os_type="linux"
 fi
 
 
-if [ "$type" == "latency" ]
+# update the version
+if [ "$os_type" == "linux" ]
```

Comment:
Shouldn't this run only on Linux?

Reply:
@coffeegoddd might be able to comment, presumably someone wanted to debug on windows. gonna resolve so I can merge